src/pages/regulations/List.vue

```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+import Table from "@/base-components/Table/Table.vue"
+import Thead from "@/base-components/Table/Thead.vue";
+import Tbody from "@/base-components/Table/Tbody.vue";
+import Tr from "@/base-components/Table/Tr.vue";
+import Td from "@/base-components/Table/Td.vue";
+import Th from "@/base-components/Table/Th.vue";
+import { useRegulationsStore } from '@/stores/regulations';
+import { computed, onBeforeMount, ref } from 'vue';
+import { useAlertStore } from "@/stores/alert";
+import Lucide from "@/base-components/Lucide/Lucide.vue";
+
+const regulationsStore = useRegulationsStore()
+const alert = useAlertStore()
+
+const isLoading = ref(true)
+
+onBeforeMount(async () => {
+    try {
+        await regulationsStore.fetchRegulations()
+        isLoading.value = false
+    }
+    catch (error) {
+        isLoading.value = false
+        alert.addErrorMessage('Yönetmelikler listelenemedi')
+
+    }
+})
+const regulations = computed(() => {
+    return regulationsStore.getRegulations
+})
+</script>
+
+<template>
+    <div class="h-screen">
```

Comment:
bu  tablo ve formları  card içine koysak çok daha güzel olacak.

http://enigma-vue.left4code.com/regular-table

bak mesela burada her bir  tabloyu ayrı ayrı card içine koymuşlar.
bizimde  layout içinde tüm içerik böyle bir card içinde olabilir
aynı yapıyı profil  sayfalarındada görebilirsin.
Özellikle edit sayfasında o anda hangi içeriği editlediğini bilmiyor kullanıcı
hangi yönetmeliği düzenliyor sayfada hiçbir bilgi yok.
card yaptığında  başlığına  o an hangi  yönetmeliği düzenliyorsa bununla ilgili bilgi yazılır

Ayrıca  yine profil ve login sayfalarında görebileceğin gibi
kaydete bastığında   butonu  disable edip  yükleniyor ikonu çıkarıyorum
bu şunun için önemli server biraz geç cevap verdiğinde peş peşe tıklamaları engellemiş oluyoruz.
ve kullanıcı yaptığı aksiyonun alındığını anlıyor.
özellikle mobilde  dokunamadımmı acaba duygusunun önüne geçiyor